Package.swift

```diff
@@ -2,14 +2,17 @@
 import PackageDescription
 
 let package = Package(
-    name: "WallE",
+    name: "WALL·E",
```

Comment:
@RuiAAPeres let me know if you agree with this rename, if so we could rename the repository as well.

Reply:
I like how it looks but won't this make it harder to use as a dependency?

Reply:
Not sure if this will be used as a dependency to be honest, this can always be changed if necessary or I can rollback this change completely.

Reply:
I agree with @ilyapuchka even if it looks nice it's not ideal for a package name to have non-ascii characters.

We'll make the bot name in Slack have the nice name but as for the package I think you should revert to `WallE`. Or use `Wall-E`.

Reply:
![image](https://user-images.githubusercontent.com/1098602/58952287-2c99ed80-878b-11e9-889d-209722095cb0.png)

Reply:
Sorry.
![Sad Wall·E](https://user-images.githubusercontent.com/216089/58954032-48f05700-8798-11e9-9355-573596e12e26.png)

Reply:
It's fine 😄 Wall-E doesn't have feelings yet so 😂